drivers/sensor/qdec_mcux/qdec_mcux.c

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) 2022, Prevas A/S
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_mcux_qdec
+
+#include <errno.h>
+#include <stdint.h>
+
+#include <fsl_enc.h>
+#include <fsl_xbara.h>
+
+#ifdef CONFIG_PINCTRL
+#include <zephyr/drivers/pinctrl.h>
+#endif /* CONFIG_PINCTRL */
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/drivers/sensor/qdec_mcux.h>
+#include <zephyr/drivers/spi.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(qdec_mcux, CONFIG_SENSOR_LOG_LEVEL);
+
+struct qdec_mcux_config {
+	ENC_Type *base;
+#ifdef CONFIG_PINCTRL
+	const struct pinctrl_dev_config *pincfg;
+#endif /* CONFIG_PINCTRL */
+	XBARA_Type *xbar;
```

Comment:
@mmahadevan108  I fixed the other compliance errors but I cannot figure out why this has inconsistent spacing..?

Reply:
@mmahadevan108 Do you want me to fiddle with it until the script does not fail or is it possible to merge because it is a false positive?

Reply:
@nashif @stephanosio, can you please help with this Compliance Failure. Seems like a false positive.